### PR TITLE
Bump version to 2018.2.0

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Review by RealOrangeOne — DISMISSED** [review]
Missing bump to `setup.py`

File: debian/changelog [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sb-vision (0) unstable; urgency=low
+sb-vision (2018.2.0) unstable; urgency=low
```
Comment (PeterJCLaw) [inline debian/changelog]: Are we going to do this to all the other packages too? I've no idea whether they depend on specific versions.

This **needs** testing before this merging.

Comment (PeterJCLaw) [inline debian/changelog]: I'm pretty sure we should _add_ to the entries in this file, not replace them.

**Review by PeterJCLaw — DISMISSED** [review]

File: debian/changelog (hunk 1 above)
Comment (Adimote) [reply debian/changelog]: I've updated and tested this :+1:

Comment (Adimote) [reply debian/changelog]: Good point

Comment (PeterJCLaw) [reply debian/changelog]: For our own future reference, could you detail somewhere what the testing procedure was?

Comment (Adimote) [reply debian/changelog]: I've documented this in https://github.com/sourcebots/wiki/wiki/Update-file-Testing-Process

Comment (Adimote) [reply debian/changelog]: Fixed in aba4106

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**